maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -199,16 +210,29 @@ public String setup(Server server, boolean isCreatePlatform, boolean isCopyDepen
     }
 
     private DistroProperties createDistroForPlatform(DistroProperties distroProperties, Server server) throws MojoExecutionException {
-        if(distroProperties == null){
+        if (distroProperties == null) {
             distroProperties = new DistroProperties(server.getServerId(), server.getPlatformVersion());
-            if(server.getDbDriver().equals(SDKConstants.DRIVER_H2)){
+            if (server.getDbDriver().equals(SDKConstants.DRIVER_H2)) {
                 distroProperties.setH2Support(true);
             }
             distroProperties.saveTo(server.getServerDirectory());
         }
         return distroProperties;
     }
 
+    private void setDebugPort(Server server) {
+        if (StringUtils.isBlank(debug) || wizard.checkYes(debug)) {
+            debug = wizard.promptForValueIfMissingWithDefault(
+                    ENABLE_DEBUGGING_DEFAULT_MESSAGE,
+                    server.getDebugPort(),
+                    "port number",
+                    "no debugging");
+        }
+        if(StringUtils.isNumeric(debug)){
```

Comment:
I don't like the fact that non numeric input for debug port is silently ignored, I guess we should do something about it, e.g. show information that non-numeric input is invalid and ask for port value again.